src/core/qgsmapsettings.h

```diff
@@ -154,12 +155,15 @@ class CORE_EXPORT QgsMapSettings
 
     //! Check whether the map settings are valid and can be used for rendering
     bool hasValidSettings() const;
-    //! Return the actual extent derived from requested extent that takes takes output image size into account
+    //! Return the extent of the visible area, see visiblePolygon()
```

Comment:
should be something like "returns the bounding box of the visible area". Also use "@see visiblePolygon" so that doxygen automatically creates a link to the visiblePolygon docs.